functions/src/aqi-calculation/purple-air-response.ts

```diff
@@ -133,7 +141,6 @@ async function purpleAirToFirestore(): Promise<void> {
         sensorDocUpdate.latitude = reading.latitude;
         sensorDocUpdate.longitude = reading.longitude;
         sensorDocUpdate.name = reading.name;
-        sensorDocUpdate.purpleAirId = reading.id;
```

Comment:
is this removed just because it's redundant? Could PA ever change a PA ID?

Reply:
This was my way of automatically changing the sensor docs to use numbers instead of strings for purpleAirId, but it's no longer necessary. The PurpleAir ID won't change in a sensor call